guides/logs-to-sematext.md

```diff
@@ -0,0 +1,182 @@
+# Send Logs to Sematext
+
+[Sematext](https://sematext.com/) is a tool for managing logs, and considered
+an alternative to Splunk, but with cheaper and more flexible [pricing](https://sematext.com/pricing).
+In this article, we present an alternative to Splunk by combining Fluentd with
+the Sematext open Elasticsearch API.
+
+![](/images/sematext-dashboard.png)
+
+
+[Elasticsearch](https://www.elastic.co/products/elasticsearch) is an
+open source search engine known for its ease of use.
+[Sematext](https://sematext.com/) runs and manages Elasticsearch
+in the cloud. You also have the option to use [Kibana](https://www.elastic.co/products/kibana)
+alongside the dashboards in the Sematext UI.
+
+
+By combining Fluentd and Sematext's managed Elasticsearch + Kibana you get
+a scalable, flexible, easy to use log management tool and search engine with an intuitive native web UI.
+You also get Kibana, if you want to use it. This provides a managed Splunk alternative,
+for a fraction of the cost.
+
+In this guide, we'll cover the installation, setup, and basic use of
+this log management solution. This setup was tested on Ubuntu
+18.04. **If you're not familiar with Fluentd**, please
+learn more about Fluentd first.
+
+## Prerequisites
+
+### Set Up Sematext
+
+You need to [sign up](https://apps.sematext.com/ui/registration) and create an App. Read more in the docs [here](https://sematext.com/docs/).
+
+### Set Up Fluentd (td-agent)
+
+In this guide you'll install td-agent, the stable release of Fluentd.
+Please refer to the guides below for detailed installation steps.
+
+-   [Debian Package](/install/install-by-deb.md)
+-   [RPM Package](/install/install-by-rpm.md)
+-   [Ruby gem](/install/install-by-gem.md)
+
+Next, install the Elasticsearch plugin for Fluentd:
+`fluent-plugin-elasticsearch`.
+
+```
+$ sudo /usr/sbin/td-agent-gem install fluent-plugin-elasticsearch --no-document
+```
+
+Now you'll configure the `td-agent` (Fluentd) to interface properly with
+Elasticsearch. Please edit `/etc/td-agent/td-agent.conf` as shown
+below:
+
+```
+# Switch to debug if you need to debug
+<system>
+  log_level debug
+</system>
+
+# get logs from syslog
+<source>
+  @type syslog
+  port 42185
+  tag syslog
+</source>
+
+# get logs from fluent-logger, fluent-cat or other fluentd instances
+<source>
+  @type forward
+</source>
+
+<filter **>
+  @type stdout
+</filter>
+
+<match syslog.**>
+  @type elasticsearch
+  host logsene-receiver.sematext.com
+  # for EU
+  # host logsene-receiver.eu.sematext.com
+  port 443
+  scheme https
+  index_name <LOGS_TOKEN>
+  <buffer>
+    @type file
+    path /tmp/fluent/es-buffer/es.all.*.buffer
+    chunk_limit_size 250k
+    flush_interval 50s
+    retry_limit 5
+    retry_wait 60
+  </buffer>
+  num_threads 1
```

Comment:
use `flush_thread_count` in `<buffer>`. `num_threads` is for v0.12.

Reply:
Got it, will edit.

Reply:
Please move it into `<buffer>` section.

```
<buffer>
  # other parameters...
  flush_thread_count 1
</buffer>
```

Reply:
@repeatedly fixed! Moved to under `<buffer>` section. :smile: